ci/build_linux.sh

```diff
@@ -19,7 +19,7 @@ cd temp
 if [ "$TRAVIS_PULL_REQUEST" = "false" ]
 then
 echo "***************************** Generating config files ****" >> $BUILDLOGS
-cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON .. >>$BUILDLOGS 2>&1;
+cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DENABLE_DEBUG_METRICS=ON .. >>$BUILDLOGS 2>&1;
```

Comment:
Please rename "-DENABLE_DEBUG_METRICS=ON" to "-DBUILD_DEBUG_METRICS=ON".  Please see comment in CMakeLists.txt for more details.